lld/test/ELF/lto/i386-global-offset-table.ll

```diff
@@ -0,0 +1,22 @@
+; REQUIRES: x86
```

Comment:
What is this test testing?

Reply:
Thanks for the review. Added a file-level comment.